apps/interactive-calibration/frameProcessor.cpp

```diff
@@ -79,7 +79,7 @@ bool CalibProcessor::detectAndParseChAruco(const cv::Mat &frame)
 
     std::vector<std::vector<cv::Point2f> > corners, rejected;
     std::vector<int> ids;
-    cv::aruco::detectMarkers(frame, mArucoDictionary, corners, ids, cv::aruco::DetectorParameters::create(), rejected);
+    cv::aruco::detectMarkers(frame, mArucoDictionary, corners, ids, cv::makePtr<cv::aruco::DetectorParameters>(), rejected);
```

Comment:
use just ` const cv::aruco::DetectorParameters&` version as it simple structure.

Reply:
Will be updated with the next PR.